js/index.js

```diff
@@ -263,4 +263,19 @@ linksAction.forEach((node) => {
     menuElement.classList.add('hidden');
     document.querySelector('body').classList.remove('overflow-hidden');
   };
-});
\ No newline at end of file
+});
+
+// Email Validation
+
+const submitForm = document.getElementById('emailv');
+submitForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const emailChecked = submitForm.elements[2];
+  const emailValue = emailChecked.value;
+  if (emailValue === emailValue.toLowerCase()) {
+    submitForm.submit();
+  } else {
+    submitForm.querySelector('.form-error')
+      .textContent = 'Email must be in lower case';
+  }
```

Comment:
- [ ] Please try to fix the email validation check, the user is still able to submit emails with both lower and uppercase letters.

- If a user submits the contact form with the email field containing uppercase characters an error message should show up.
- If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

Reply:
Hi @CollinsTatang,
Me and my partner @sntakirutimana72  checked the code It's working fine.

Reply:
Please can we have a call?

Reply:
yes sure, I send you an invitation at zoom.

Reply:
Ok

Reply:
I have not received it please share here

Reply:
https://microverse.zoom.us/j/93398300976?pwd=TUVDSGRKbVkrQzlxWHdoL1N0cjRCZz09

Reply:
All required changes have been solved.
The project is currently working as required on both ends.
Thanks for your time @MoRaad97

Reply:
Thanks, @CollinsTatang 
It was very nice to meet you and chat.
And thanks for your advice.